pkg/agent/datamodel/defaults.go

```diff
@@ -29,36 +29,6 @@ type PropertiesDefaultsParams struct {
 	PkiKeySize int
 }
 
-// SetPropertiesDefaults for the container Properties
-func (cs *ContainerService) SetPropertiesDefaults(params PropertiesDefaultsParams, cloudSpecConfig *AzureEnvironmentSpecConfig) error {
```

Comment:
I think we can remove the whole defaults.go?

Reply:
Yes, we can. I have just sent this new PR to remove the defaults. https://github.com/Azure/AgentBaker/pull/315